packages/@tinacms/cli/src/buildTina/index.ts

```diff
@@ -167,6 +167,7 @@ export const build = async ({
   skipIndex,
 }: BuildOptions) => {
   const rootPath = ctx.rootPath as string
+  const tinaGeneratedPath = path.join(rootPath, '.tina', '__generated__')
```

Comment:
should this be down below the rootPath null check?

Reply:
Yes, fixed.